src/features/shared/sliding/SlidingItem.tsx

```diff
@@ -45,19 +47,54 @@ const OptionContainer = styled.div<{ active: boolean }>`
     `}
 `;
 
+const custom_slash_lengths: Dictionary<number> = {
+  [bookmark]: 35,
+  [mailUnread]: 40,
+};
+
+const SlashedIcon = styled(IonIcon)<{
+  icon: string;
+  slash: boolean;
+  bgColor: string;
+}>`
+  ${({ icon, slash, bgColor }) =>
+    slash &&
+    css`
+      &::after {
+        content: "";
+        position: absolute;
+        height: ${custom_slash_lengths[icon] ?? 30}px;
+        width: 3px;
+        background: var(--ion-color-${bgColor}-contrast);
+        font-size: 1.7em;
+        left: 50%;
+        top: 50%;
+        transform: translate(-50%, -50%) rotate(-45deg);
+        transform-origin: center;
+        box-shadow: 0 0 0 2px var(--ion-color-${bgColor});
+      }
+    `}
+`;
+
 export type SlidingItemAction = {
   /**
    * If `string`, it's passed to IonIcon as an icon value
    */
-  render: (() => React.ReactNode) | string;
+  icon: string;
   trigger: () => void;
   bgColor: string;
+  slash?: boolean;
 };
 
+export type ActionList = [
```

Comment:
Can we add some docs regarding when the first and second items represents (near and far swipe respectively)